go/vt/sqlparser/ast.go

```diff
@@ -2484,21 +2484,6 @@ type (
 		Fsp  int // fractional seconds precision, integer from 0 to 6 or an Argument
 	}
 
-	// ExtractedSubquery is a subquery that has been extracted from the original AST
```

Comment:
Removing this struct was a big part of the refactoring. Before this refactoring, we would replace all subqueries with this AST struct early on in the semantic analysis process, and use it during planning. All of that logic is now gone from semantic analysis and this struct is no longer needed.